removed "rustup" & added new runner tag

fevo1971 opened the pull request:
* added new tag **windows** for the new windows-runner(s)
* removed the *rustup* comand from build-script. rustversion & target installed via ansible
  (https://gitlab.parity.io/parity/devops/blob/master/ansible/roles/win-gitlab-runner/tasks/main.yml)

gabreal commented:
this will switch the windows build to the new azure windows 10 runners - in case something goes wrong one can simply apply the windows tag to the old runners :+1:

5chdn commented on .gitlab-ci.yml:
```diff
@@ -162,6 +163,7 @@ test-windows:
     - sh scripts/gitlab/test-all.sh stable
   tags:
    - rust-windows
+   - windows
```
what's the difference?

5chdn replied:
I mean, why two tags, just tag the new runners with rust-windows and fine?

General-Beck commented:
new tags for runners are planned, more standard and understandable. like `docker`, `darwin`, `windows` & `shell`

fevo1971 commented:
ok, i changed the tag to (just) **windows** and added the new tag to the runner settings of *betelgeuse* and *bellatrix* so we can managed the migration from the old to the new runners nodes in gitlab instead of changing the *gitlab-ci.yml* in the future.

5chdn commented:
Hehe, but why aren't we just applying the rust-windows tag to the new runners? This way we won't need any changes to the current gitlab-ci YML? :)

ddorgan commented:
@5chdn good to go now? The tag name was reverted. This is required to test the new windows CI hosts..
